lib/package/components/navbar.css

```diff
@@ -2,9 +2,12 @@
 
 /* Navbar Components */
 .navbar {
+  /* position: sticky;
```

Comment:
You can remove these commented codes if they do not need.

Reply:
> You can remove these commented codes if they do not need.

Yeah I forgot. Thanks for noting